features/checkout/paying_for_order/seeing_payment_method_instructions_on_thank_you_page.feature

```diff
@@ -11,11 +11,19 @@ Feature: Seeing payment method instructions on thank you page
         And the store has a payment method "Offline" with a code "OFFLINE"
         And it has instructions "Account number: 0000 1111 2222 3333"
 
-    @ui
+    @ui @no-api
     Scenario: Being informed about payment instructions on thank you page
         Given I am a logged in customer
         And I have product "PHP T-Shirt" in the cart
         When I proceed selecting "Offline" payment method
         And I confirm my order
         Then I should see the thank you page
         And I should be informed with "Offline" payment method instructions
+
+    @api @no-ui
+    Scenario: Being informed about payment instructions
+        Given I am a logged in customer
+        And I have product "PHP T-Shirt" in the cart
+        When I proceed selecting "Offline" payment method
+        And I confirm my order
+        And I should be informed with "Offline" payment method instructions
```

Comment:
I'm wondering whether the information about the "thank you" page is important here. Maybe we can go with

```suggestion
    @ui @api
    Scenario: Being informed about payment instructions on thank you page
        Given I am a logged in customer
        And I have product "PHP T-Shirt" in the cart
        When I proceed selecting "Offline" payment method
        And I confirm my order
        Then I should be informed with "Offline" payment method instructions
```

wdyt?
cc: @GSadee @NoResponseMate

Reply:
I'm fine with that, but swap the `on thank you page` from the scenario/feature/filename to `after checkout`

Reply:
Ah, yes, I missed that.